DebugView++/MainFrame.cpp

```diff
@@ -268,7 +268,7 @@ std::wstring FormatDuration(double seconds)
 	if (minutes > 0)
 		return wstringbuilder() << FormatUnits(minutes, L"minute") << L" " << FormatUnits(floor_to<int>(seconds), L"second");
 
-	static const wchar_t* units[] = { L"s", L"ms", L"�s", L"ns", nullptr };
+	static const wchar_t* units[] = { L"s", L"ms", L"µs", L"ns", nullptr };

```

Comment:
I seems to compile and work just fine, so no harm done.

On 12 March 2015 at 22:39, justanotheranonymoususer <
notifications@github.com> wrote:

> In DebugView++/MainFrame.cpp
> <https://github.com/djeedjay/DebugViewPP/pull/170#discussion_r26347085>:
>
> > @@ -268,7 +268,7 @@ std::wstring FormatDuration(double seconds)
> >  	if (minutes > 0)
> >  		return wstringbuilder() << FormatUnits(minutes, L"minute") << L" " << FormatUnits(floor_to<int>(seconds), L"second");
> >
> > -	static const wchar_t* units[] = { L"s", L"ms", L"�s", L"ns", nullptr };
> > +	static const wchar_t* units[] = { L"s", L"ms", L"µs", L"ns", nullptr };
>
> Hmm, the encoding of µ changed to UTF-8. Perhaps this should be reverted.
> Sorry.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/djeedjay/DebugViewPP/pull/170/files#r26347085>.
>



-- 
Met vriendelijke groeten,

Jan Wilmans